frontend/src/components/profile/components/edite-document-popup/edite-document-popup.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { IDocument } from 'common/interfaces';
+import { Modal, Button, Radio } from 'components/common';
+import { ButtonColor, ButtonStyleType, DocumentStatus, ButtonType } from 'common/enums';
+import { BindingCb } from 'common/types';
+import { useForm } from 'react-hook-form';
+import {
+  documentStatusToReadable,
+  EditeDocumentCb,
+  DocumentFormData,
+  DocumentFormKey,
+} from '../../common';
+import { createOptions } from 'helpers';
+import styles from './styles.module.scss';
+
+type Props = {
+  document: IDocument;
+  isShow:boolean;
+  onToggleModal:BindingCb;
+  onEditeDocument:EditeDocumentCb;
+};
+
+const verifyDocumentOptions = createOptions<string>(Object.values(DocumentStatus), (documentStatus) => ({
+  value: documentStatus,
+  label: documentStatusToReadable[(documentStatus as DocumentStatus)],
+}));
+
+const EditeDocumentPopup: React.FC<Props> = ({ document, isShow, onToggleModal, onEditeDocument }) => {
+  const defaultValues = { [DocumentFormKey.STATUS]: document.status };
```

Comment:
```suggestion
  const DEFAULT_VALUES = {
    [DocumentKey.STATUS]: document.status,
  };
```
Move it to the `frontend/src/components/profile/components/edite-document-popup/common/constants`